source/features/html-preview-link.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {groupSiblings} from '../libs/group-buttons';
+
+function init(): void {
+	const rawButton = select('#raw-url') as HTMLAnchorElement;
+	if (rawButton && /\.html?/.test(rawButton.pathname)) {
+		const link = rawButton.pathname.split('/');
+		// Remove /raw/
+		link.splice(3, 1);
+		rawButton
+			.parentElement! // `BtnGroup`
+			.prepend(
+				<a
+					className="btn btn-sm BtnGroup-item"
+					href={`https://cdn.statically.io/gh${link.join('/')}`}>
+					Preview
+				</a>
+			);
+		groupSiblings(rawButton);
```

Comment:
This probably isn’t necessary in this feature. Maybe it’s not necessary even in the feature you copied it from, but I don’t remember.

Reply:
You mean `groupSiblings`? It's from `copy-file` feature. Will check whether it is needed later 😊

Reply:
You're right #2506